pym/_emerge/BinpkgExtractorAsync.py

```diff
@@ -39,6 +46,21 @@ def _start(self):
 			self._async_wait()
 			return
 
+		decompression_binary = varexpand(shlex_split(decomp_cmd)[0], mydict=self.env)
```

Comment:
Good catch, i thought it was defined in actions.py as well, but there's only a fallback for PORTAGE_BZIP2_COMMAND.